src/modules/TeamsList/components/Teams/components/TeamUserTable/components/TableRow/index.tsx

```diff
@@ -23,14 +23,18 @@ export const TableRow: FC<TableRowProps> = ({
   const { firstName, lastName, score, telegram, discord, github, country, city, id } = member;
   const dispatch = useDispatch();
   return (
-    <tr className={secondTable ? 'SecondTable' : 'FirstTable'}>
-      <TableCell value={count.toString(10)} />
+    <tr
+      className={
+        secondTable ? `SecondTable ${!count && 'MentorRow'}` : `FirstTable ${!count && 'MentorRow'}`
```

Comment:
you can create separate variables for template strings for better readability

Reply:
fixed